.github/actions/upload/action.yml

```diff
@@ -5,6 +5,10 @@ inputs:
     default: ${{ github.job }}
   test-report-retention-days:
     default: 5
+  heap-dump-retention-days:
+    default: 2
```

Comment:
Can we afford 7 here? or 14?
2 is pretty low value

(i know that heap dumps are big)

Reply:
I think a higher value should be fine since this doesn't happen that often. I'll update this to 14.